### PR TITLE
Rewrite to RoaveDeveloperTools

### PR DESCRIPTION
**Pull request by Ocramius**
This PR is a full rewrite from 0, providing a better/more flexible overall architecture and hopefully output as well.

### CODE REVIEW

File: src/Roave/DeveloperTools/Inspector/TimeInspector.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (Danielss89) [inline src/Roave/DeveloperTools/Inspector/TimeInspector.php]: I think you want the license information here instead?

File: src/Roave/DeveloperTools/Inspector/TimeInspector.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        $this->inspections[spl_object_hash($event)] = [$microtime, $microtime];
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function inspect(EventInterface $event)
+    {
+        $microtime = microtime(true);
+        $oid       = spl_object_hash($event);
```
Comment (Danielss89) [inline src/Roave/DeveloperTools/Inspector/TimeInspector.php:54]: I like full names, so i would use $objectId instead of $oid

Comment (Ocramius) [inline src/Roave/DeveloperTools/Inspector/TimeInspector.php:54]: I will likely get rid of this, as the `TimeInspector` seems pretty much useless (on its own) to me

Comment (Ocramius) [inline src/Roave/DeveloperTools/Inspector/TimeInspector.php:54]: TBD in #6

**Comment by Ocramius** [comment]
This is going to be merged as "good enough"

**Pull request merged**